src/main/java/dk/kb/netarchivesuite/solrwayback/parsers/WarcParser.java

```diff
@@ -253,7 +254,7 @@ public static ArcEntry getWarcEntryZipped(String warcFilePath, long warcEntryPos
   }
 
     
-    public static BufferedInputStream lazyLoadBinary(String arcFilePath, long arcEntryPosition) throws Exception{
+    public static BufferedInputStream lazyLoadBinary(String arcFilePath, long arcEntryPosition, long binarySize) throws Exception{
```

Comment:
Dont think it is necessary to add binarySize to the API.
Both the warc and arc parser re-parse the header to the get file-pointer to the correct position. And in this
parsing there binarySize is calculated again.
Sp same issue in lazyloadBinary method in ArcParser.